src/main/java/com/libertymutualgroup/herman/aws/ecs/broker/rds/AuroraClient.java

```diff
@@ -62,7 +62,7 @@
 
 import static com.libertymutualgroup.herman.aws.ecs.broker.rds.RdsBroker.pollingIntervalMs;
 
-public class AuroraClient implements RdsClient {
+public class AuroraClient extends StandardRdsClient {
```

Comment:
Not sure if we have any test projects that use Aurora or not. StandardRdsClient brings in a bunch of logic that wouldn't apply to Aurora, right @c1phr?

Reply:
AuroraClient overrides every method in StandardRdsClient so I figured it'd be fine

Reply:
mainly did this to avoid replicating the big setDefaults method I pulled into the clients since it appeared to be the same as the standard ones for aurora

Reply:
Ok, all good then